CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## About VelaUX
 
-The [KubeVela](https://github.com/oam-dev/kubevela) User Experience (UX) Dashboard, it provides out-of-box application delivery and management platform.
+The [KubeVela](https://github.com/oam-dev/kubevela) User Experience (UX) Platform, it provides out-of-box application delivery and management platform.
```

Comment:
we already explain it's a platform at the end of the sentence, user experience platform seems weird.

Reply:
I will rewrite the project description with the new PR.